r/src/main/java/org/molgenis/r/service/RExecutorServiceImpl.java

```diff
@@ -79,7 +79,7 @@ public void loadTable(
       if (variables.isEmpty()) {
         execute(
             format(
-                "is.null(base::assign('%s', value={arrow::read_parquet('%s')}))",
+                "is.null(base::assign('%s', value={data.frame(arrow::read_parquet('%s', as_data_frame = FALSE))}))",
```

Comment:
ah, this makes sense actually

Reply:
Haha, well, I thought it didn't make sense, `as_data_frame = FALSE` combined with `data.frame` is total madness, but it works.